### PR TITLE
[13.0][IMP] logistics_planning_base: draft button + new icons

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
- New button Mark as Draft
- Adds new icons in state buttons

### CODE REVIEW

**Comment by ChristianSantamaria** [comment]
@dalonsod Installed in test enviroment, can u review?

File: logistics_planning_invoicing/__manifest__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
     ''',
     'author': 'Solvos',
     'license': 'LGPL-3',
-    'version': '13.0.2.0.1',
+    'version': '13.0.2.2.0',
```
Comment (dalonsod) [inline logistics_planning_invoicing/__manifest__.py]: These changes should be applied in `logistics_planning_base` manifest file.

File: logistics_planning_base/views/logistics_schedule_views.xml [hunk 2]
```diff
@@ -248,26 +258,36 @@
         <field name="arch" type="xml">
             <form>
                 <header>
+                    <button
+                        name="%(action_logistics_schedule_draft)d"
+                        type="action"
+                        string="Mark as Draft"
+                        attrs="{'invisible': [('state', '=', 'draft')]}"
```
Comment (dalonsod) [inline logistics_planning_base/views/logistics_schedule_views.xml]: Desired flow for `draft` state as destination is only from `cancel` state:
```suggestion
                        attrs="{'invisible': [('state', '!=', 'cancel')]}"
```

File: logistics_planning_base/models/logistics_schedule.py [hunk 3]
```diff
@@ -295,6 +298,13 @@ def action_logistics_schedule_finished(self):
     def action_logistics_schedule_done(self):
         self.browse(self.env.context.get("active_ids", []))._action_done()
 
+    def _action_draft(self):
+        to_draft = self.filtered(lambda x: x.state in ['ready', 'done', 'cancel'])
```
Comment (dalonsod) [inline logistics_planning_base/models/logistics_schedule.py]: Desired flow for `draft` state as destination is only from `cancel` state:
```suggestion
        to_draft = self.filtered(lambda x: x.state == 'cancel')
```

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments, when attended, I'll test it ASAP

**Comment by ChristianSantamaria** [comment]
@dalonsod updated and tested in test enviroment :+1:

**Review by dalonsod — APPROVED** [review]
Tested :+1:

**Pull request merged**